src/features/instance/log/index.tsx

```diff
@@ -86,28 +78,34 @@ const isValidDateRange = (startDate?: Date, endDate?: Date) => {
 };
 
 export function Logs() {
-	const [logFilters, setLogFilters] = useState<z.infer<typeof LogFiltersSchema>>({
-		limit: 1000,
-		order: 'desc',
+	const [logFilters, setLogFilters] = useState<z.infer<typeof LogFiltersFormSchema>>({
+		limit: '1000',
```

Comment:
What do you think about defaulting this to 100? 1000 feels a bit large, my browser starts to lag with it a bit. A mobile browser would get impacted even more.

Reply:
I was thinking the exact same thing tbh.